src/assets/js/main.js

```diff
@@ -858,6 +858,7 @@ if (!Function.prototype.bind) {
     init: function() {
       var self = this;
       $('#filter-dataset-form #q').on('keyup', function(event) {
+        console.log(event.keyCode)
```

Comment:
This is like me leaving print() everywhere ;) Can we remove it next time we touch that file, no need to resubmit this PR.